packages/theming/atlas/src/themesource/atlas_core/web/main.scss

```diff
@@ -311,7 +311,7 @@
     @include dijit-widget();
 }
 
-@import "core/widgetscustom/switch-widget";
+@import "core/widgets/switch";
 @if not $exclude-custom-switch {
```

Comment:
There is no default value for this variable yet. Do we also want to include it in the new theme folder, so that users who migrate later have the latest and greatest variables in their theme folder.

Reply:
Ah, thanks for the reminder. I will add this 👍

Reply:
I'm confused. There is a corresponding variable in `theme`.

Reply:
I have just seen now that you reuse an already defined exclusion variable. Sorry for the confusion. We already have this in `theme/web/exclusion-variables.scss`. I do have a concern here if we need to support older widgets with Atlas 3.

Reply:
As far as I know, there is no way to have Switch 3.0.0 and Switch 2.X in the same project as there's an `id` clash. 

However if end-users have created their own widgets following Switch 2.X HTML structure, [this](https://github.com/mendix/widgets-resources/blob/c42159c1e5ce29188d134469098e1e1880478770/packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_switch.scss#L54-L150) and [this](https://github.com/mendix/widgets-resources/blob/c42159c1e5ce29188d134469098e1e1880478770/packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_switch.scss#L171-L207) code in Atlas 3 will support these cases (as it's backwards compatible).

Note: the comment's suppose to say "// below is maintained for backwards compatibility prior to Switch **3.0.0**."

Reply:
Switch 3 should be for Atlas 3 and Mendix 9. 
Switch 2 should be for Atlas 2.

Reply:
@cjfhodges Okay, that's clear. But what about including CSS in Atlas 3 that supports user-defined widgets that follow the HTML structure of Switch 2.0.0?

Reply:
If we don't, any projects with user-defined widgets or HTML similar to Switch 2.0.0 HTML structure will break. IMO there's value in keeping it and little "noise" in having it in the Atlas files; net plus.

Reply:
I think we shouldn't take user-defined widgets into account here. These styles were solely meant for the switch widget, right? We can ofcourse take into consideration to not break HTML structure too much to make migration of any custom styles from users on the switch widget less painful.

Reply:
It's been decide that Atlas 3 needs to support Switch 2. See [this](https://mendix.slack.com/archives/G01KNRM8A5C/p1618213070000700?thread_ts=1617800916.007500&cid=G01KNRM8A5C) thread.